vuepress/docs/next/tutorials/compose/content-types-tutorial.md

```diff
@@ -58,9 +62,9 @@ This adds the configured Attribute to the Attribute Type.
 
 ![modifyContentType](./img/content_types4.png)
 
-- (Optional) Complete the additional configuration steps required by your Attribute Type, if applicable (the 'Text' Attribute Type requires no additional configuration). 
+- Complete the additional configuration steps required by your Attribute Type, if applicable (the 'Text' Attribute Type requires no additional configuration). 
 
-- (Optional) Add other Attribute Types to your Attribute, subject to the above configuration process.
+- (Optional) Add other Attribute Types to your Content Type, subject to the above configuration process.
```

Comment:
subject to the above configuration process >> subject to the configuration process above